projects/packages/my-jetpack/_inc/constants.ts

```diff
@@ -4,7 +4,7 @@ export const MY_JETPACK_MY_PLANS_PURCHASE_NO_SITE_SOURCE = 'my-jetpack-my-plans-
 export const MY_JETPACK_PRODUCT_CHECKOUT = 'my-jetpack-product-checkout';
 
 export const MyJetpackRoutes = {
-	Home: '/',
+	Home: '/:tab',
```

Comment:
Should we be more exhaustive here which options for tabs are available? As I understand, currently it might match anything other than defined paths?